servicepulse/intro-pending-retries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Pending Retries Message Management in ServicePulse
```

Comment:
Isn't pending just one stage of retrying a message. If so, can we have the title as "Retrying failed messages in Service Pulse" and have the pending status and explanations for it be part of this? The title seems to be very implementation specific and not sure how friendly it is to the user.